### DISCUSSION TITLE
Infobar APIs

### DISCUSSION THREAD

**Discussion by skysteve** [elgatosf/elgato#122] [Help]
Hey, I know the stream deck neo is still pretty new so hopefully this is already in the works, but any ETA on APIs for the infobar? Feels really limiting at the moment that it's only a clock/page indicator. Apologies if I've missed it in the docs/samples but I can't find anything :) Thanks in advance!

**Comment by skysteve** [comment]
cc @ChekTek since I see you've been replying to other posts on here 🙏

**Comment by ChekTek** [comment]
Hi @skysteve!

The team has been discussing unlocking this, but unfortunately it's lower on the priority list at the moment.

**Reply by skysteve** [reply]
Thanks for the reply, feels weird it's low priority given it's the newest product but 🤷

**Comment by skysteve** [comment]


**Reply by ChekTek** [reply]
Sadly no, I'm afraid the plans to add this feature have not been added to the roadmap at this time. 😔

**Reply by skysteve** [reply]
😞  Thanks for the reply but sad news. Would be really cool to be able to build things for that screen, hopefully soon! There's so many possibilities there

**Comment by ChekTek** [comment]
Closing this issue, as it has been reported to the Stream Deck team as a feature request.
